src/library/masq.js

```diff
@@ -89,6 +105,22 @@ class Masq {
     this.masterKey = null
     this.peer = null
     this.app = null
+    this.dbName = null // used only during userApp registration
+
+    // init state
+    this.state = STATES.NOT_LOGGED
+  }
+
+  setState (newState) {
+    if (STATE_DEBUG) console.log(` ##### From ${this.state} -> ${newState} ######`)
```

Comment:
can you wrap this line in a `debug` function like in masq-lib ?

Reply:
yes done